pkg/apis/eventing/v1alpha1/broker_types.go

```diff
@@ -113,6 +114,12 @@ type BrokerStatus struct {
 	//
 	// It generally has the form {broker}-router.{namespace}.svc.{cluster domain name}
 	Address duckv1alpha1.Addressable `json:"address,omitempty"`
+
```

Comment:
I wonder if we should hide this in a implementation detail struct in the status. like:

`obj.status.implv1.TriggerChannel/IngressChannel` (but with a better name)

because I realize you need this data, it also breaks the broker blackbox a little.

Reply:
addressable is how you send events to the broker. Trigger/Ingress channels are internal deets so trigger knows how to wire things up. So, I think we need all three.

Reply:
oh, I see, I thought you were talking about Address field based on the comment :)